lib/db/mysql.js

```diff
@@ -155,7 +156,7 @@ module.exports = function (log, error) {
 
   // Insert : accounts
   // Values : uid = $1, normalizedEmail = $2, email = $3, emailCode = $4, emailVerified = $5, kA = $6, wrapWrapKb = $7, authSalt = $8, verifierVersion = $9, verifyHash = $10, verifierSetAt = $11, createdAt = $12, locale = $13
```

Comment:
Thanks for noticing @philbooth. I'm often blind to code comments. It'd be nice if these were auto-generated. The intent was to save one from having to search for the proc to see the args, but maintaining them is tedious and easy to forget. I'll fix this up but I'm open to discussing their removal.